### DISCUSSION TITLE
support config numa for  ceph osd?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12949] [Q&A]
releated doc https://ceph.io/en/news/blog/2023/reef-osds-per-nvme/

**Comment by travisn** [comment]
`numactl`? We don't have any rook docs on using it.

**Reply by zhucan** [reply]
Is there a plan to support it?

**Reply by travisn** [reply]
Rook is always open to contributions

**Reply by 0xAlcibiades** [reply]
https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/ - there may be some k8s native ways to do this.
